Bixal/move.mil
# Validate phone numbers

Thanks for submitting a pull request! Below are a few things you can do to help us more quickly review your changes.

## Checklist

I have…

      against my changes

## Summary of Changes

This pull request…

- Updates Drupal
- Verify phone number is not empty when reading the XML file

## For the Drupal update
Execute `drush entity-updates`

## Testing

To verify the changes proposed in this pull request…

1. UPload XML file
1. Go to the locator maps tool and look for "Wright Patterson AFB"
1. You should see the results.


## Screenshots
<img width="678" alt="Screen Shot 2019-03-26 at 1 37 14 AM" src="https://user-images.githubusercontent.com/37001835/54979063-c6614500-4f67-11e9-9f4d-62c5d7f52192.png">
